corehq/toggles.py

```diff
@@ -2127,6 +2127,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
+EXPRESSION_REPEATER = StaticToggle(
+    'expression_repeater',
+    'Integrate with generic APIs using UCR expressions',
+    TAG_SOLUTIONS_OPEN,
```

Comment:
UCR itself is `TAG_SOLUTIONS_LIMITED`, and I think this feature has the possibility to do dangerous/non-performant things in the same fashion. Perhaps better to start more restrictive?

Reply:
Updated, although that UCR tag is going to change soon.